File: Java/Arreglos3.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        char[] arr = {'A', 'B', 'C', 'D', 'E'};
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingresa la letra a buscar: ");
        char valor = sc.next().charAt(0);
        boolean encontrado = false;

        for(int i = 0; i < arr.length; i++) {
            if(arr[i] == valor) {
                System.out.println("Encontrado en la posición " + i);
                encontrado = true;
                break;
            }
        }

        if(!encontrado)
            System.out.println("No se encontró la letra.");
    }
}

File: C++/Arreglos3.cpp
#include <iostream>
using namespace std;

int main() {
    char arr[] = {'A', 'B', 'C', 'D', 'E'};
    int n = sizeof(arr) / sizeof(arr[0]);
    char valor;
    bool encontrado = false;

    cout << "Ingresa la letra a buscar: ";
    cin >> valor;

    for(int i = 0; i < n; i++) {
        if(arr[i] == valor) {
            cout << "Encontrado en la posición " << i << endl;
            encontrado = true;
            break; // Detener el bucle si encontramos el valor
        }
    }

    if(!encontrado)
        cout << "No se encontró la letra." << endl;

    return 0;
}




